pkg/operation/botanist/logging.go

```diff
@@ -162,7 +162,7 @@ func (b *Botanist) DefaultVali() (vali.Interface, error) {
 		v1beta1constants.PriorityClassNameShootControlPlane100,
 		nil,
 		b.ComputeValiHost(),
-		true,
+		false,
```

Comment:
With this, the `authEnabled` parameter is set to `false` in all invocations of this `shared.NewVali` function. Can you please remove it from the signature?

Reply:
ok